utils/publish/publish-snapshot.sh

```diff
@@ -94,19 +89,6 @@ pre_publish_check() {
     fi
 }
 
-create_npmrc() {
```

Comment:
Creating the `.npmrc` file was only for setting the auth token. Now with trusted publishing, we no longer need that.